Src/UIComponents/ProfileInfo/ProfileInfo.ts

```diff
@@ -17,32 +18,31 @@ function profileInfo(profileData: {
     height: number, // cm
```

Comment:
I found that you use here 'createStyledElement' function (line 7) that could be replaced by that that we have in utils.

Reply:
Right 👍 Corrected